README.md

```diff
@@ -25,7 +25,7 @@ nix-env -f channel:nixpkgs-unstable -iA tmatrix
 TMatrix uses the ncurses library.
 To install the library on Ubuntu or Debian run:
 ```
-sudo apt-get install libncurses5-dev
+sudo apt-get install libncurses5-dev libncurses5
```

Comment:
libncurses5-dev isn't actually needed. The dev version was added by mistake so you can remove it.
You can retitle the commit to `Fixed the required ncurses library in the README.md`.